src/main/java/nextstep/subway/DataLoaderConfig.java

```diff
@@ -28,12 +29,12 @@ public void run(String... args) throws Exception {
         Station 양재역 = new Station("양재역");
         Station 남부터미널역 = new Station("남부터미널역");
 
-        Line 신분당선 = new Line("신분당선", "red lighten-1", 강남역, 양재역, 10);
-        Line 이호선 = new Line("2호선", "green lighten-1", 교대역, 강남역, 10);
-        Line 삼호선 = new Line("3호선", "orange darken-1", 교대역, 양재역, 10);
+        Line 신분당선 = new Line("신분당선", "red lighten-1", 강남역, 양재역, new Distance(10));
```

Comment:
```suggestion
        Line 신분당선 = new Line("신분당선", "red lighten-1", 강남역, 양재역, 10);
```

Line 생성하는 클라이언트 입장에서 Distance라는 클래스의 존재를 모르면서 생성할 수 있도록 해봐도 좋겠네요 ㅎㅎ